src/functions/defaults.js

```diff
@@ -0,0 +1,24 @@
+exports.command = ({ conf, help, run }, file) => {
```

Comment:
What if you default both conf and help? Also, if we have more exports (let's say, `exports.test = someFunction`), this will remove it. Same issue was addressed in Klasa by adding the new properties with `this.prop = someProp`, but this one will make exports that are not `conf`, `help` or `run` (init is not included) be private.

Reply:
conf and help are already both defaulted